README.md

```diff
@@ -39,7 +39,7 @@ That's it! Now you can enjoy super fast `nuxt dev` experience with Vite!
 
 **[📖 Read documentation for more](https://vite.nuxtjs.org)**
 
-## ✔️ What is working?
+## ✔️ What is working and tested?
```

Comment:
Adding tested at current stage is not correct since they features are basically working but not covering all edge-cases